cmd/conformance-tester/run-e2e-tests.sh

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###
+### This script can be used to run a whole range of conformance tests against
+### a single KKP seed cluster. Use this to verify somewhat quickly that all
+### provider/OS/release combinations work as expected.
+###
+### $ PROVIDERS=aws,azure,openstack RELEASES=1.25,1.26 ./conformance-tester.sh
+###
+
+set -euo pipefail
+
+echodate() {
+  echo "[$(date -Is)] $@"
+}
+
+###########################################################
+# configuration
+###########################################################
+KUBECONFIG="${KUBECONFIG:-}"
```

Comment:
Irrrrrrrrrrrrrrrrrrrghs, I don't know if we want to ship this shitty script of mine as part of the official image. With every KKP release we have to adjust a bit here and there and I really, really do not want to get support requests because something in this script doesn't fit someone's usecase.

Reply:
I kinda thought you will say that. Let me try to add preset/credentials-loading capability to the conformance-tester. The script is not doing much more.

Reply:
Can't you just keep this script in your repo?

Reply:
True. Will move it to where we trigger the conformance tests.